docs/organizer-guide/meetups/starting.rst

```diff
@@ -224,3 +224,16 @@ After Your Meetup
 * Send a thank you note to your host.
 * Post pictures on your meetup page. Be sure to get permission.
 * Use Twitter to thank your attendees, speakers and sponsor.
+
+
+If You Can't Find a Speaker
+---------------------------
+
+It is not always easy to find a speaker. Other meetups have done well with
+alternatives:
+
+* Coffee Klatches; suitable for meetups with a few people.
+* View a Write the Docs Conference Talk, available from our video library.
+Some meetups have combined this with live Q&A with the speaker
+* Pure networking meetup; you can "roundtable" a specific topic; give each
+attendee a chance to talk about what they do
```

Comment:
Looks like someone already added some info about remote speakers. The following paragraph appears after the list of alternatives to speakers:

If you can't find a local speaker, find a video of a talk from a previous Write the Docs conference http://www.writethedocs.org/conf/. Ask the speaker(s) if they're willing to field questions. Share the video with the group during the meetup. Set up questions and answers with the speaker during the Meetup. For one successful example, see the following Meetup ...

So a suggested revision:

Remove the text you added above.

Move the paragraph about "if you can't find a local speaker..." to the end of the "Speaker" subsection.

Suggested rewording:

If you can't find a local speaker, consider screening a popular talk from one of the
Write the Docs conferences <http://www.writethedocs.org/videos/>_ and inviting the conference presenter to call in for a live Q & A session. Both the Boulder and Austin meetups had good luck with this format.

    So you need to document an API <https://www.meetup.com/Boulder-Denver-WriteTheDocs-Meetup/events/232962552/>_ remotely to provide live Q & A about the talk.
    Living documentation<https://www.meetup.com/WriteTheDocs-ATX-Meetup/events/231859187/>_

If you choose this format, make sure your meeting venue supports high quality video presentation and phone conferencing.

Or you could add the remote speaker format at the end of the list of alternatives with the details following.

Actually, the other format is to just host a remote speaker with no conference talk like Kristof is doing for the London meetup: https://www.meetup.com/Write-The-Docs-London/events/236170664/